clients/fides-js/src/lib/tcf/i18n/tcf-i18n-utils.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * TCF-specific i18n utility functions. These are isolated to a separate file so
+ * that Rollup can easily remove this code from the base fides.js bundle without
+ * having to do any particularly difficult tree-shaking!
```

Comment:
OK so we can't share the non-tcf `loadMessagesFromFiles` method from `i18n-utils.ts` because of difficulties with Rollup, I remember running into this before, with trying to share methods between non-tcf and tcf